docs/experiments-msmarco-v2.md

```diff
@@ -0,0 +1,61 @@
+# Anserini: Guide to Working with the MS MARCO V2 Collections
+
+Indexing the passage collection:
+
+```
+sh target/appassembler/bin/IndexCollection -collection MsMarcoPassageV2Collection \
+ -generator DefaultLuceneDocumentGenerator -threads 18 \
+ -input collections/msmarco_v2_passage \
+ -index indexes/msmarco-passage-v2 \
+ -storePositions -storeDocvectors -storeRaw
+```
+
+Adjust `-threads` as appropriate.
+The above configuration, on a 2017 iMac Pro with SSD, takes around 30min.
+
+The complete index occupies 72 GB (138,364,198 passages).
+It's big because it includes postions (for phrase queries), document vectors (for relevance feedback), and a complete copy of the collection itself.
+Index size can be reduced by removing the options `-storePositions`, `-storeDocvectors`, `-storeRaw` as appropriate.
+
+Indexing the document collection:
+
+```
+sh target/appassembler/bin/IndexCollection -collection MsMarcoDocV2Collection \
+ -generator DefaultLuceneDocumentGenerator -threads 18 \
+ -input collections/msmarco_v2_doc \
+ -index indexes/msmarco-doc-v2 \
+ -storePositions -storeDocvectors -storeRaw
+```
+
+Same instructions as above.
+On the same machine, indexing takes around 40min.
+Complete index occupies 134 GB (11,959,635 documents).
+
+Perform a run on the dev queries:
```

Comment:
What about dev2

Reply:
Setting aside dev2 until we get further guidance on its "proper use".